core/src/main/resources/application.conf

```diff
@@ -103,10 +103,12 @@ workflow-options {
   //workflow-failure-mode: "ContinueWhilePossible"
 }
 
-// Optional call-caching configuration. Disabled by default.
+// Optional call-caching configuration.
 call-caching {
   enabled = false
 
+  // This option determines whether the name of a docker image should be hashed (if docker is specified for a task)
+  hash-docker-names = true
```

Comment:
We should sort out these details in TTs not PR comments. 

For example, we don't want to always hash file paths because the contents might match even when the paths are different.